docs/DEVELOPING.md

```diff
@@ -186,6 +186,154 @@ $ make ftest NAME=s3
 
 ℹ️ The e2e test uses default values defined in [configure Amazon S3 connector](#configure-amazon-s3-connector)
 
+## Google Cloud Storage Connector
+
+The [Elastic Google Cloud Storage connector](https://github.com/elastic/connectors-python/blob/main/connectors/sources/gcs.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **technical preview**_.
```

Comment:
Documentation for S3, Network Drive, ABS, GCS and Generic Database should document these connector clients as available in Beta starting in 8.7.